app/helpers/organisation_helper.rb

```diff
@@ -2,4 +2,31 @@ module OrganisationHelper
   def child_organisations_count(organisation)
     organisation["works_with"].values.flatten.count
   end
+
+  def child_organisation_count(organisation)
+    child_orgs = organisation.content_item.content_item_data["links"]["ordered_child_organisations"]
+    child_orgs.present? ? child_orgs.count : 0
+  end
+
+  def show_organisation(organisation)
+    if organisation.content_item.content_item_data["details"]["organisation_govuk_status"]["status"] == "live"
+      render partial: 'show_organisation',
+             locals: { organisation: @organisation }
+    else
+      render partial: 'separate_website',
+             locals: { organisation: @organisation }
+    end
+  end
+
+  def ordered_featured_policies(organisation)
+    featured_policies = []
```

Comment:
This feels like it could be a bit simpler.  It seems to be just making a new array from another array, or default to an empty array if it doesn't exist.
It also seems very similar to a lot of things in the `organisation` model, so it feels like it might belong there? 
Perhaps:
```
def ordered_featured_policies(organisation)
  organisation.content_item.content_item_data["links"]["ordered_featured_policies"].presence || []
end
```
might work?  Check that you've got a test that covers it before changing anything.

Reply:
I have changed it and added tests for empty or present featured policies in the content store and it's passing 👍